.changes/1.25.5.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "category": "``dms``",
+    "description": "This release adds DMS Fleet Advisor APIs and exposes functionality for DMS Fleet Advisor. It adds functionality to create and modify fleet advisor instances, and to collect and analyze information about the local data infrastructure.",
+    "type": "api-change"
+  },
+  {
+    "category": "``iam``",
+    "description": "Documentation updates for AWS Identity and Access Management (IAM).",
+    "type": "api-change"
+  },
+  {
+    "category": "``m2``",
+    "description": "AWS Mainframe Modernization service is a managed mainframe service and set of tools for planning, migrating, modernizing, and running mainframe workloads on AWS",
+    "type": "api-change"
+  },
+  {
+    "category": "``neptune``",
+    "description": "This release adds support for Neptune to be configured as a global database, with a primary DB cluster in one region, and up to five secondary DB clusters in other regions.",
+    "type": "api-change"
+  },
+  {
+    "category": "``redshift-serverless``",
```

Comment:
Looks like we'll need to remove all the `redshift-serverless`-related things, similar to https://github.com/boto/botocore/pull/2692